tests/blanket-options.js

```diff
@@ -0,0 +1,19 @@
+/* globals blanket, module */
+
+var options = {
+  modulePrefix: 'ember-frost-notifier',
+  filter: '//.*ember-frost-notifier/.*/',
+  antifilter: '//.*(tests|template).*/',
+  loaderExclusions: [],
+  enableCoverage: true,
+  cliOptions: {
```

Comment:
No worries, don't spend too much time on it, we've got #2 to track it. Don't let it hold this up.